pkg/cri/resource-manager/cache/utils.go

```diff
@@ -267,6 +267,7 @@ func findContainerDir(podCgroupDir, podID, ID string) string {
 	cpusetDir := cgroups.Cpuset.Path()
 
 	dirs = []string{
+		path.Join(cpusetDir, podCgroupDir, ID),
```

Comment:
What is the kubelet and containerd configuration wrt. the cgroup driver in this case ? IOW, is this the real correct `containerd, cgroupfs` in all cases ? If it is then is the 3rd entry after inserting this wrong and unnecessary ? If not, when is the 3rd entry correct instead of this ?